docs/usage/reading-list.md

```diff
@@ -0,0 +1,74 @@
+# Reading list
+
+Renovate's documentation has a lot of pages.
+To ease you into using Renovate we created reading lists.
+The reading lists contain the essential information for each type of user.
+
+How much you should read depends on how much you want to customize Renovate's behavior.
+
+## How to use this page
+
+We created reading lists for these types of users:
+
+- Beginners
+- Intermediate
+- Advanced
+
+Start with the "Beginners" reading list.
+If you're self-hosting or need to update private packages, complete the relevant reading lists for those.
+
+## I don't know where to start
+
+If you're new to Renovate, you should:
+
+- Use the Mend-hosted GitHub App, or let somebody else self-host Renovate for you
+- Stick with the `config:base` or `config:best-practices` preset
+- Use the Dependency Dashboard (`config:base` or `config:best-practices` enables it automatically)
+- Read the pages in the "Beginners" list
+- Only create custom Renovate configuration when really needed
+
+## Beginners
+
+Start by reading:
+
+- [Installing & Onboarding](./getting-started/installing-onboarding.md)
+- [Key concepts, Dependency Dashboard](./key-concepts/dashboard.md)
+- [Use Cases](./getting-started/use-cases.md)
+- [Running Renovate](./getting-started/running.md)
+- [Troubleshooting](./troubleshooting.md)
+- [Known limitations](./known-limitations.md)
+- [Release notes for major versions](./release-notes-for-major-versions.md)
+
+## Intermediate
+
+First, complete the "Beginners" reading list.
+Read this list _after_ experiencing Renovate's default behavior, once you really want/need to make changes to Renovate's behavior.
+
+- [Key concepts, presets](./key-concepts/presets.md)
+- [Key concepts, Renovate scheduling](./key-concepts/scheduling.md)
+- [Key concepts, automerge](./key-concepts/automerge.md)
+- [Key concepts, pull requests](./key-concepts/pull-requests.md)
+- [Noise Reduction](./noise-reduction.md)
+
+Skim the [repository configuration options](./configuration-options.md) to learn about the kind of customizations you can make to Renovate.
+Feel free to read up on anything that looks intersting to you.
+
+## Advanced
+
+First, complete the "Beginners" and the "Intermediate" reading list.
+Then read:
+
+- Define your own regex manager with [`regexManagers`](./configuration-options.md#regexmanagers)
```

Comment:
```suggestion
- Define your own regex manager with [`regexManagers`](./configuration-options.md#regexmanagers)
- [Shareable config presets](./config-presets) 
```

Let's put shareable config presets in the list for "Advanced" users. It's one of the key selling points for Renovate, so we should probably include it.